src/components/Header.tsx

```diff
@@ -44,13 +45,13 @@ function Header(props: HeaderProps) {
 					name="twitter:image"
 					content="https://res.cloudinary.com/degyjrpjj/image/upload/v1688584020/website/share/ugaacimkpgt6rsnecvds.png"
 				/>
-				{/* FIX: 🤔 how to add analytics in a Nextjs project? */}
-				<script
-					src="https://beamanalytics.b-cdn.net/beam.min.js"
-					data-token={process.env.BEAM_DATA_TOKEN}
-					async
-				></script>
 			</Head>
+			{/* FIX: 🤔 how to add analytics in a Nextjs project? */}
+			<Script
+				src="https://beamanalytics.b-cdn.net/beam.min.js"
+				data-token={process.env.BEAM_DATA_TOKEN}
+				async
+			/>
```

Comment:
Is this working now? We'll have to see the deployment I guess.

Reply:
I want to test it again that's why I put it back outside of the Head